src/components/SingleArticle.jsx

```diff
@@ -22,6 +22,11 @@ function SingleArticle() {
 			link: '',
 		},
 	]
+	function handleArticleVote(value) {
+		voteArticle({ inc_votes: value }, article_id).then((data) => {
+			setArticle(data)
+		})
+	}
```

Comment:
This does work great, however, you should also consider what the user should see if the vote was unsuccessful. I see that the number doesn't change, however, I would also recommend displaying a little message for the user if the vote is unsuccessful. For example if the user is offline